acceptance-tests/tests/src/test/resources/crypto/secp256r1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "chainId": 1981,
+    "ecCurve": "secp256r1",
+    "constantinoplefixblock": 0,
+    "ethash": {
+      "fixeddifficulty": 1000
+    }
+  },
+  "nonce": "0x0",
+  "timestamp": "0x58ee40ba",
+  "extraData": "0xf83ea00000000000000000000000000000000000000000000000000000000000000000d594c2ab482b506de561668e07f04547232a72897daf808400000000c0",
```

Comment:
You can remove the extraData from the genesis file because it is not needed for ethhash.

Reply:
:+1: